dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/FirstOrderWithDelay.mo

```diff
@@ -0,0 +1,59 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FirstOrderWithDelay "Delay with a first order filter smoothing the input signal"
+
+  import Modelica;
+  import Dynawo;
+
+  extends Modelica.Blocks.Interfaces.SISO;
+
+  parameter Dynawo.Types.Time delayTime "Delay time of output with respect to input signal";
```

Comment:
for all time parameters and variables do we want to precise the unit? or do we consider that because it is a block i can be used with several units?

Reply:
"in s" is written in the description of every time parameter. The branch can be merged.